spec/javascripts/components/shared/icon_spec.js

```diff
@@ -0,0 +1,21 @@
+import jasmineEnzyme from 'jasmine-enzyme';
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import Icon from 'components/shared/Icon';
+
+describe('<Icon />', function() {
+
+  beforeEach(jasmineEnzyme);
+
+  it("should be material icon", function() {
+    const wrapper = shallow(<Icon material='check_circle' />);
+    expect(wrapper.find('i').html()).toBe('<i class="mi ">check_circle</i>');
```

Comment:
Actually I was expecting it to behave like this :) The reason is to make [<Icon />](https://github.com/Codeminer42/cm42-central/pull/141/files#diff-ba00482c0f39ab1824b6e144777ef0a6R5) simpler..